fdbserver/BulkLoadUtil.actor.cpp

```diff
@@ -92,32 +95,34 @@ ACTOR Future<std::string> readBulkFileBytes(std::string path, int64_t maxLength)
 			}
 		}
 
-		return content;
+		return *content;
 	} catch (Error& e) {
 		TraceEvent(SevWarn, "ReadBulkFileBytesError").error(e).detail("Path", path).detail("MaxLength", maxLength);
 		throw;
 	}
 }
 
-ACTOR Future<Void> writeBulkFileBytes(std::string path, StringRef content) {
+ACTOR Future<Void> writeBulkFileBytes(std::string path, StringRef contentIn) {
 	try {
+		state std::string contentString = contentIn.toString();
+		state std::shared_ptr<std::string> content = std::make_shared<std::string>(std::move(contentString));
```

Comment:
is this copy needed?

Reply:
I have another PR to remove these copies. This PR is just a quick fix to unblock simulation errors.